tests/phpunit/tests/post/output.php

```diff
@@ -90,7 +90,7 @@ function test_the_content_shortcode() {
 		$this->assertSame( strip_ws( $expected ), strip_ws( get_echo( 'the_content' ) ) );
 	}
 
-	function test_the_content_shortcode_paragraph() {
+	public function test_the_contentshortcode_paragraph() {
```

Comment:
@costdev The name for this test is confusing. But renaming it is out of scope for this ticket.

Reply:
Actually, this was an error in my underscore removal. A partial match on `_shortcode_paragraph()` removed this underscore by mistake. I've corrected this and will be force pushing momentarily.